src/reference/config/writing-platform-configs.rst

```diff
@@ -49,277 +49,282 @@ Why Were Platforms Introduced?
 
 - Allow a compute cluster with multiple login nodes to be treated as a single
   unit.
-- Allow Cylc to elegantly handle failure of to communicate with login nodes.
-- Configure multiple platforms with the same hosts; for example you can use
-  separate platforms to submit jobs to a batch system and to background on
-  ``localhost``.
+- Allow Cylc to elegantly handle failure to communicate with login nodes.
+- Reduce the number of ssh connections required for job submission and polling.
 
 .. _Install Targets:
 
 What Are Install Targets?
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Install targets represent file systems. More than one platform can use the
-same file system. It defaults to the name of the platform.
+same file system. Cylc relies on the site setup file ``global.cylc`` to determine
```

Comment:
I think we use "config" or "configuration" elsewhere. IMO "setup" is better, but consistency is better still.
```suggestion
same file system. Cylc relies on the site configuration file ``global.cylc`` to determine
```